src/pages/Home/Home.tsx

```diff
@@ -1,17 +1,40 @@
 import { Button } from '@mui/material';
+import Modal from '@src/components/Modal/Modal';
+import React, { useState } from 'react';
+
+const HomePage: React.FC = () => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
```

Comment:
React.FC is redundant here 
```suggestion
const HomePage = () => {
  const [isModalOpen, setIsModalOpen] = useState(false);
```